dags/openshift_nightlies/manifest.yaml

```diff
@@ -182,6 +182,40 @@ platforms:
           install: rogcp/ovn.json
           benchmarks: data-plane.json
 
+  aro:
+    versions: ["4.10", "4.11"]
+    variants:
+      - name: sdn-control-plane
+        schedule:  "00 13 * * 3"
+        config:
+          install: azure/sdn-cp.json
+          benchmarks: control-plane.json
+      - name: ovn-control-plane
```

Comment:
@Harshith-umesh did you test an OVN install? I don't see anything in the install code that captures which SDN to use. Additionally, with ARO I think you still need to jump through some hoops for OVN unless I'm mistaken.

Reply:
Good find, I can install this [wheel](https://aka.ms/az-aroext-latest.whl) and pass in `--sdn-type`

```
    --sdn-type --software-defined-network-type    : SDN type either "OpenShiftSDN" (default) or
                                                    "OVNKubernetes".  Allowed values: OVNKubernetes,
                                                    OpenShiftSDN.
```

Reply:
More Info on installing the preview is [here](https://learn.microsoft.com/en-us/azure/openshift/howto-configure-ovn-kubernetes#install-and-use-the-preview-azure-command-line-interface-cli)

Before and after installing the extension changes in parameter is [here](https://gist.github.com/krishvoor/83646514cca5baf58d6ad17549a9b2f3)